onnxruntime/test/optimizer/qdq_test_utils.h

```diff
@@ -91,6 +91,46 @@ GetQDQTestCaseFn BuildQDQConvTransposeTestCase(const std::vector<int64_t>& input
   };
 }
 
+// Creates the graph:
+// quantized_input -> DequantizeLinear -> InstanceNormalization -> QuantizeLinear -> quantized_output.
+// Currently used to test QNN EP.
+template <typename QuantType>
+GetQDQTestCaseFn BuildQDQInstanceNormTestCase(const std::vector<int64_t>& input_shape, float epsilon) {
```

Comment:
Builds the following graph programmatically.
![DQ_InstanceNorm_Q_onnx_model](https://user-images.githubusercontent.com/19691973/222839947-c4629abc-3c61-480c-af44-0dcf746e599b.png)